js/initalize/layers.populate.js

```diff
@@ -0,0 +1,189 @@
+// Layering
+const imageCollection = layers.registerCollection(
+	"image",
+	{w: 2560, h: 1536},
+	{
+		name: "Image Layers",
+	}
+);
+
+const bgLayer = imageCollection.registerLayer("bg", {
+	name: "Background",
+});
+const imgLayer = imageCollection.registerLayer("image", {
+	name: "Image",
+});
+const maskPaintLayer = imageCollection.registerLayer("mask", {
+	name: "Mask Paint",
+});
+const ovLayer = imageCollection.registerLayer("overlay", {
+	name: "Overlay",
+});
+const debugLayer = imageCollection.registerLayer("debug", {
+	name: "Debug Layer",
+});
+
+const imgCanvas = imgLayer.canvas; // where dreams go
+const imgCtx = imgLayer.ctx;
+
+const maskPaintCanvas = maskPaintLayer.canvas; // where mouse cursor renders
```

Comment:
Here I expose the mask layers with the same names as before, so using them as in the past should not be an issue.